src/zerocoin_v3.cpp

```diff
@@ -750,8 +750,9 @@ void CSigmaState::Containers::RemoveMint(sigma::PublicCoin const & pubCoin) {
     mint_info_container::const_iterator iter = mintedPubCoins.find(pubCoin);
     if (iter != mintedPubCoins.end()) {
         mintMetaInfo[iter->second.coinGroupId][iter->second.denomination] -= 1;
+        CMintedCoinInfo tmpMintInfo(iter->second);
```

Comment:
This will slower than the previous version due to it need to construct `CMintedCoinInfo`. Is there any reason why we do this?

Reply:
@ultimaweapon the copy is done because erase call below can invalidate iterator